Sources/Parsing/ParserPrinters/TryMap.swift

```diff
@@ -0,0 +1,90 @@
+extension Parser {
+  /// Returns a parser that tries to transforms the output of this parser with a given closure.
+  ///
+  /// Unlike the standard ``map(_:)-4hsj5`` function, the `transform` may throw an error, which is passed on if it occurs during a `parse`.
+  ///
+  /// This method is similar to `Sequence.map`, `Optional.map`, and `Result.map` in the Swift
+  /// standard library, as well as `Publisher.map` in the Combine framework.
+  ///
+  /// ## Printability
+  ///
+  /// Mapping a parser with a transform function results in a parser that cannot print. If you need
+  /// to transform the output of a parser and retain printability, use the ``Conversion``-based
+  /// ``map(_:)-2sblf`` operator instead.
+  ///
+  /// - Parameter transform: A closure that transforms values of this parser's output.
+  /// - Returns: A parser of transformed outputs.
+  ///
+  ///
+  @_disfavoredOverload
```

Comment:
```suggestion
```

This is only necessary on operations that are also present on `Sequence`, since `String` is both a parser and a sequence.

Reply:
Ah ok. I'm not familiar with the annotation, so just left it in place after copy-pasting from `Map`.